scripts/templates/init-defaultworker.template

```diff
@@ -9,4 +9,6 @@ userdel hydro-service \
  && chmod -R 3777 /hs_tmp /shared_tmp /tmp 
 
 celery beat -A hydroshare -s /hydroshare/celery/celerybeat-schedule &
-celery worker -A hydroshare -E -Q default
+celery worker -A hydroshare -E -Q default --concurrency=4 &
```

Comment:
@sblack-usu It might be better to make ```4``` as a parameter in hydroshare-config.yml since I believe it corresponds to the number of cores available in the VM that runs defaultworker container and may vary depending on the specific deployment environment.

Reply:
@hyi - I'm unsure how to put this in config.  This is what I tried, but it failed.

The above line
`celery worker -A hydroshare -E -Q default --concurrency=${CELERY_CONCURRENCY} &`

The line in hydroshare-config.yml
`CELERY_CONCURRENCY: 4`

Do you know how to do this?

Reply:
@sblack-usu Here is what is done to have environment variables loaded in hsctl script: https://github.com/hydroshare/hydroshare/blob/develop/hsctl#L20-L21

Reply:
Got it, thanks @hyi